docs/sdk/serverless-signing/eip191.md

```diff
@@ -1,58 +1,48 @@
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# Signing with EIP191
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-Use EIP191 `eth_personal_sign` to sign a message instead of a transaction or raw signature.
+# Signing within a Lit Action using EIP-191
 
-This can be done with
+## Overview
 
-```js
-LitActions.ethPersonalSignMessageEcdsa({ message, publicKey , sigName }); 
-```
+Lit Actions offer the ability to use [EIP-191](https://eips.ethereum.org/EIPS/eip-191?ref=blog.spruceid.com) to sign a message instead of a transaction or raw signature. This is done with the Lit Actions method [ethPersonalSignMessageEcdsa](https://actions-docs.litprotocol.com/#ethpersonalsignmessageecdsa). 
 
-It will prepend `"\x19Ethereum Signed Message:\n"` to the message and then hash and sign it according to https://eips.ethereum.org/EIPS/eip-191.
+It will prepend `"\x19Ethereum Signed Message:\n"` to the message and then hash and sign it according to the EIP-191 standard.
 
-Lit Action code:
+## Prerequisites
 
-```js
-export const litActionCode = `
-(async () => {
-  const sigShare = await LitActions.ethPersonalSignMessageEcdsa({
-    message: dataToSign,
-    publicKey,
-    sigName,
-  });
-})();
-`;
-```
+- Basic understanding of [Lit Actions](../serverless-signing/quick-start)
+
+## Complete Code Example
+
+The complete code example is available in the [Lit Developer Guides Code Repository](https://github.com/LIT-Protocol/developer-guides-code/tree/master/eip-191-signing). There you can find a Node.js and browser implementation of this example code.
+
+### Example Lit Action
 
-Lit SDK code:
-
-```js
-const litActionSignatures = await litNodeClient.executeJs({
-  sessionSigs,
-  code: litActionCode,
-  jsParams: {
-    dataToSign: ethersUtils.arrayify(
-      ethersUtils.keccak256([1, 2, 3, 4, 5])
-    ),
-    publicKey: pkpPublicKey,
-    sigName: "sig",
-  },
-});
-
-const dataSigned = `0x${signature.dataSigned}`;
-const encodedSig = ethersUtils.joinSignature({
-  v: signature.recid,
-  r: `0x${signature.r}`,
-  s: `0x${signature.s}`,
-});
-
-const recoveredPubkey = ethersUtils.recoverPublicKey(dataSigned, encodedSig);
-console.log("Recovered uncompressed public key: ", recoveredPubkey);
-
-const recoveredAddress = ethersUtils.recoverAddress(dataSigned, encodedSig);
-console.log("Recovered address from signature: ", recoveredAddress);
+The following Lit Action uses `ethPersonalSignMessageEcdsa` to append `"\x19Ethereum Signed Message:\n"` to the given message and then hash and sign it according to the EIP-191 standard. The `sigShare` variable is set to a boolean value of `true` if the signature was successfully generated, and `false` otherwise. The Lit Action will also return the complete signature as an object under the `sigName` key.
```

Comment:
```suggestion
The return value of `ethPersonalSignMessageEcdsa` (the `sigShare` variable in this example) is set to a boolean value of `true` if the signature was successfully generated, and `false` otherwise. The Lit Action will also return the complete signature as an object under the `sigName` key.
```